mne/gui/_coreg.py

```diff
@@ -723,8 +724,13 @@ def _redraw(self, verbose=None):
             helmet=self._add_helmet,
         )
         with self._redraw_mutex:
-            logger.debug(f'Redrawing {self._redraws_pending}')
-            for key in self._redraws_pending:
+            # We need at least "head" before "hsp", because the grow_hair param
+            # for head sets the rr that are used for inside/outside hsp
+            redraws_ordered = sorted(
+                self._redraws_pending,
+                key=lambda key: list(draw_map).index(key))
```

Comment:
FYI this is a bugfix we should keep regardless of whether or not the octant approach in this PR is used/useful